2-extra/1-choose-your-own-adventure.js

```diff
@@ -53,6 +58,8 @@ let game = {
     //
     // Hint: the room objects have north/east/south/west methods which return
     // a new room object that is in the relevant direction.
+    //this.currentRoom = this.currentRoom[direction]();
+    this.currentRoom = this.currentRoom[direction]();
```

Comment:
Will this always work? What happens in `this.currentRoom[direction]` is `null` or `undefined`?